tests/integration/test_utils.go

```diff
@@ -258,7 +261,7 @@ func setup(t *testing.T) {
 
 func sync(kongFile string, opts ...string) error {
 	deckCmd := cmd.NewRootCmd()
-	args := []string{"sync", "-s", kongFile}
+	args := []string{"gateway", "sync", kongFile}
```

Comment:
We are keeping the "old" syntax on purpose here since we care more about not breaking existing workflows than introducing possible bugs into new functionalities. @Tieske has some WIP task to improve our posture here.